app/formSchema/pages/submission.ts

```diff
@@ -26,6 +22,10 @@ const submission = {
         title: 'Title',
         type: 'string',
       },
+      submissionDate: {
+        title: 'On this date (YYYY-MM-DD)',
+        type: 'string',
+      },
```

Comment:
Is this change needed?

Reply:
I prefer to keep it in the same order as it appears on the form itself or the `ui:schema`, it's more readable that way.